### PR TITLE
Add fax to contact points

### LINKED ISSUE #2695: BUG: Add missing fax number to Provider Contact (Encounter Info)
## Description

The eCR below is missing its fax number in the Provider contact info.

Potential problem:
- In `_ContactPoint.liquid`, we only add the telecom if `ContactPoint.value` contains `tel:`. Instead, in this example eCR, the telecom value we're missing contains `fax:`.

Potential solution:
- Add a conditional in `_ContactPoint.liquid` that includes fax numbers.

Make sure that:
- [ ] Fax number is added in `_ContactPoint.liquid` (and make sure it conforms to the R4 FHIR spec)
- [ ] When multiple contact numbers are available, that all numbers show up
- [ ] Any relevant tests are added

## Steps to reproduce

Example eCR: `5ff5928a-6c29-4a0c-bbe5-c96484bbd22c`
- [Missing facility contact information in encounter info section](https://skylight-hq.slack.com/archives/C07DX0PCCSJ/p1724271006674419)

## Actual result

From [5ff5928a-6c29-4a0c-bbe5-c96484bbd22c](http://localhost:3000/view-data?id=1.2.840.114350.1.13.202.3.7.8.688883.309#facility-details):

## Priority

_Provide an estimate for level of impact the bug has today by placing an X in the applicable bracket_

#### Priority
- [ ] **Low priority** _The bug does not have to be fixed immediately. High and medium priority bugs should be addressed first._
- [x] **Medium priority** _Typically, medium-priority bugs do not affect customers directly and therefore can be fixed in the normal course of testing and development._
- [ ] **High priority** _High priority bugs must be addressed immediately. They often affect the app’s functionality and impact customers and their user experience. They must take priority._

## Notes

Will require some FHIR conversion.

### PR DESCRIPTION
**Pull request by mcmcgrath13**
# PULL REQUEST

## Summary
Make sure the fax number is converted along with other telecom info. Depends on https://github.com/skylight-hq/FHIR-Converter/pull/4

* add to snapshot test for converter service
* expect `fax` system usage in `formatContactPoint`
* add unit test

## Related Issue
Fixes #2695

## Additional Information
Before:
image

With this change:
image

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/fhir-converter/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
 
 # Download FHIR-Converter
-RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch mcm/customize --single-branch /build/FHIR-Converter
```
Comment (mcmcgrath13) [inline containers/fhir-converter/Dockerfile:4]: TODO: Don't merge until this points to a released version

File: containers/ecr-viewer/src/app/services/formatService.tsx [hunk 2]
```diff
@@ -581,6 +581,13 @@ export const formatContactPoint = (
       );
     } else if (contactPoint.system === "email" && contactPoint.value) {
       contactArr.push(contactPoint.value);
+    } else if (contactPoint.system === "fax" && contactPoint.value) {
+      const faxNumberUse = toSentenceCase(contactPoint.use ?? "");
+      contactArr.push(
+        [faxNumberUse, "Fax", formatPhoneNumber(contactPoint.value ?? "")]
```
Comment (mcmcgrath13) [inline containers/ecr-viewer/src/app/services/formatService.tsx:587]:
```suggestion
        [faxNumberUse, "Fax:", formatPhoneNumber(contactPoint.value ?? "")]
```

File: containers/ecr-viewer/src/app/tests/services/formatService.test.tsx [hunk 3]
```diff
@@ -519,6 +519,21 @@ describe("formatContactPoint", () => {
...
+        value: "+13135551234",
+        use: "work",
+      },
+      {
+        system: "fax",
+        value: "+1 313 555-1235",
+      },
+    ];
+    const actual = formatContactPoint(contactPoints);
+    expect(actual).toEqual(["Work Fax 313-555-1234", "Fax 313-555-1235"]);
```
Comment (mcmcgrath13) [inline containers/ecr-viewer/src/app/tests/services/formatService.test.tsx:535]:
```suggestion
    expect(actual).toEqual(["Work Fax: 313-555-1234", "Fax: 313-555-1235"]);
```

**Review by BobanL — APPROVED** [review]

File: containers/fhir-converter/Dockerfile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
 
 # Download FHIR-Converter
-RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch v7.0-skylight-3 --single-branch /build/FHIR-Converter
+RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch mcm/ecr-provider-fax --single-branch /build/FHIR-Converter
```
Comment (mcmcgrath13) [inline containers/fhir-converter/Dockerfile:4]:
```suggestion
RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch v7.0-skylight-5 --single-branch /build/FHIR-Converter
```

**Pull request merged**